src/carriers/portmonitor_carrier/MonitorBinding.cpp

```diff
@@ -25,8 +29,14 @@ MonitorBinding::~MonitorBinding()
 
 MonitorBinding* MonitorBinding::create(const char* script_type)
 {
-    if(ConstString(script_type) == "lua")
+    if(ConstString(script_type) == "lua") {
+#ifdef ENABLED_PORTMONITOR_LUA
         return new MonitorLua();
+#else
+        yError()<<"Lua portmonitor plugin is not enabled!";
```

Comment:
Perhaps this error should be a little more specific, as "Lua portmonitor plugin is not enabled!" is a bit cryptic... something like "Cannot load LUA plugin. YARP portmonitor was built without LUA support".